internal/checker/types.go

```diff
@@ -63,6 +63,7 @@ const (
 	TypeFormatFlagsUseSingleQuotesForStringLiteralType TypeFormatFlags = 1 << 28 // Use single quotes for string literal type
 	TypeFormatFlagsNoTypeReduction                     TypeFormatFlags = 1 << 29 // Don't call getReducedType
 	TypeFormatFlagsOmitThisParameter                   TypeFormatFlags = 1 << 25
+	TypeFormatFlagsWriteCallStyleSignature             TypeFormatFlags = 1 << 26 // Write construct signatures as call style signatures
```

Comment:
Modifying `TypeFormatFlags` should also require updating `nodebuilder.Flags` (we are missing the comment on `TypeFormatFlags` from Strada that says to keep them in sync).

Reply:
Strada directly calls the lower level helper that `SignatureToString` uses. Didn't want to expose more methods, thought it was better to just have another format flag.